src/sysupdate/meson.build

```diff
@@ -13,6 +13,16 @@ systemd_sysupdate_sources = files(
         'sysupdate.c',
 )
 
+systemd_sysupdated_sources = files(
+        'sysupdated.c',
+)
```

Comment:
For only one file this seems unnecessary? Just inline `files()`?

Reply:
done